script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -92,19 +92,19 @@ I'm not sure if my mind can handle this whole situation of having lunch with Cie
 So I'd prefer to sit in a place where we can't be seen by many people.
 Looking around the cafeteria, I see that there are empty seats in the middle while the corner seats are already occupied by other students' belongings.
 At this hour, there are no corner seats for two people―――
-"―――There it is."
+"―――There it is."//to be rechecked, 「―――、あった」
```

Comment:
"There it is" he was looking for them and he saw them

Reply:
"Ah, perfect." works for me

Reply:
The context is that the only remaining place is occupied by Arihiko, who he sort of doesn't want to sit with cos that would interrupt romance time with Ciel. That's why there's a ――― pause before. He spots Arihiko and goes 'ah shit'.

Reply:
> Oh ――― there's one.

I think that works best after thinking about it for a while